packages/expo-notifications/android/src/main/java/expo/modules/notifications/notifications/scheduling/NotificationScheduler.kt

```diff
@@ -197,6 +198,23 @@ open class NotificationScheduler : Module() {
         )
       }
 
+      "monthly" -> {
+        val day = params["day"] as? Number
+        val hour = params["hour"] as? Number
+        val minute = params["minute"] as? Number
+
+        if (day == null || hour == null || minute == null) {
+          throw InvalidArgumentException("Invalid value(s) provided for yearly trigger.")
```

Comment:
the error message could contain the provided values for better debuggability

Reply:
Let's review all these error messages and update them in a separate PR.